src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/tests/VideoInstructionsLink.unit.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { expect } from 'chai';
+import moment from 'moment';
+import { renderWithStoreAndRouter } from '~/platform/testing/unit/react-testing-library-helpers';
+import { MockAppointment } from '../../../../tests/mocks/unit-test-helpers';
+import { VIDEO_TYPES } from '../../../../utils/constants';
+import VideoInstructionsLink from '../VideoInstructionsLink';
+
+describe('VAInstructionsLink component', () => {
+  it('should display instructions for video visit', async () => {
+    // Arrange
+    const mockAppointment = new MockAppointment({ start: moment() });
+    mockAppointment.setComment('Video Visit Preparation');
+    mockAppointment.setKind(VIDEO_TYPES.mobile);
+
+    // Act
+    const screen = renderWithStoreAndRouter(
+      <VideoInstructionsLink appointment={mockAppointment} />,
+      {
+        initialState: {},
+      },
+    );
```

Comment:
You can replace `renderWithStoreAndRouter` with `render` since you are not passing any params with value other than an empty initialState
```suggestion
        const screen = render(
      <VideoInstructionsLink appointment={mockAppointment} />,
    );
```

Reply:
Done.